packages/valory/skills/abstract_round_abci/base.py

```diff
@@ -78,6 +79,15 @@
 TransactionType = TypeVar("TransactionType")
 
 
+def get_name(prop: Any) -> str:
```

Comment:
https://github.com/valory-xyz/open-autonomy/pull/1642#discussion_r1045711425
-> property typing doesn't work

Reply:
(As in it does work but it doesn't yield the desired effect. Mypy detects a different type than `property` and so it all becomes very cumbersome)